packages/teams-js/src/artifactsForCDN/validDomains.json

```diff
@@ -29,9 +29,7 @@
     "edgeservices.bing.com",
     "www.bing.com",
     "www.staging-bing-int.com",
-    "teams.cloud.microsoft",
```

Comment:
Can we make a note somewhere of the specific cloud domains that we want to allowlist as well. Just in case we need to back out this wildcard at some point. It would be useful to document the domains we want too.

Reply:
Sure, where would you like me to put that note? JSON doesn't support comments. Also note that there are some cloud.microsoft domains that we need to support now but which I can't put in a note in this repo because their existence is not public knowledge.

Reply:
Maybe our internal wiki?

Reply:
I was thinking we could document all wildcards in the Wiki, but I'm open to other suggestions. I am also ok that it is the unit test.